data/autoyast_opensuse/autoyast_btrfs_quota.xml

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+    <bootloader>
+        <global>
+            <timeout config:type="integer">-1</timeout>
+        </global>
+    </bootloader>
+    <general>
+        <mode>
+            <confirm config:type="boolean">false</confirm>
+        </mode>
+    </general>
+    <login_settings>
+        <autologin_user>bernhard</autologin_user>
+    </login_settings>
+    <networking>
+        <keep_install_network config:type="boolean">true</keep_install_network>
+    </networking>
+    <software>
+        <install_recommended config:type="boolean">true</install_recommended>
+        <products config:type="list">
+            <product>openSUSE</product>
+        </products>
+        <patterns config:type="list">
+            <pattern>apparmor</pattern>
```

Comment:
Do we need those patterns? I would suggest to avoid adding unnecessary packages/patterns as they get renamed, etc. and break the tests.

Reply:
Removed.